src/funding/applepay/config.jsx

```diff
@@ -25,7 +25,8 @@ export function getApplePayConfig() : FundingSourceConfig {
             BUTTON_LAYOUT.VERTICAL
         ],
 
-        Logo: ({ logoColor, optional }) => ApplePayLogo({ logoColor, optional }),
+        Logo:  ({ logoColor, optional }) => ApplePayLogo({ logoColor, optional }),
+        Marks: ({ ...props }) => ApplePayMark({ ...props }),
```

Comment:
Can we just call this `Mark` since there's only one?

Reply:
No, there's two.

<---

Reply:
![onlyone](https://user-images.githubusercontent.com/1219075/112684455-4f9a0380-8e30-11eb-907e-abd6aa748333.gif)